erizo/src/erizo/WebRtcConnection.cpp

```diff
@@ -620,37 +631,31 @@ void WebRtcConnection::updateState(TransportState state, Transport * transport)
 }
 
 // changes the outgoing payload type for in the given data packet
-void WebRtcConnection::queueData(int comp, const char* buf, int length, Transport *transport,
-                                packetType type, uint16_t seqNum) {
+void WebRtcConnection::queueData(std::shared_ptr<dataPacket> packet) {
```

Comment:
:+1: to this, having the information in the packet is cleared, especially sequence number information.